native/or1200/or1200-cbasic.c

```diff
@@ -1,6 +1,7 @@
 /* Test basic c functionality, and BSS section clearing.  */
-#include "cpu-utils.h"
-#include "lib-utils.h"
+//#include "cpu-utils.h"
```

Comment:
Remove the commented lines completely perhaps?

Reply:
Yes, I will. This old code is pretty messy.  Once it's all working I'll do a once over cleanup.

Thanks for having a look. There is more to come for the mmu test which is still broken.